packages/api/src/external/cda/generate-ccd.ts

```diff
@@ -32,7 +32,7 @@ export async function generateCcd({
 
   return `<ClinicalDocument xmlns="urn:hl7-org:v3" xmlns:sdtc="urn:hl7-org:sdtc" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" moodCode="EVN">
 	<realmCode code="US"/>
-	<typeId root="2.16.840.1.113883.1.3" extension="POCD_HD000040"/>
+  <typeId root="2.16.840.1.113883.1.3" extension="POCD_HD000040"/>
```

Comment:
intentional to remove the spaces?

Reply:
No. Not sure why it's like that. In my IDE, it shows normally.. :/